gluon/gluon-mesh-batman-adv-core/files/lib/gluon/upgrade/mesh-batman-adv-core/invariant/020-wireless

```diff
@@ -22,6 +22,7 @@ local function configure_radio(radio, index, config)
 		mode = 'ap',
 		ssid = config.ssid,
 		macaddr = util.generate_mac(2, index),
+		disabled = config.ssid_disabled,
```

Comment:
This is completely broken, it will overwrite the setting made by the user on each upgrade.